src/testing_system/internal/controller/server.py

```
@@ -47,12 +47,12 @@ async def get_progress():
         return orchestrator.progress
 
     @app.get("/experiments", response_model=List[Experiment])
-    async def list_experiments():
+    async def list_experiments(k: int = Query(..., description="Количество последних экспериментов")):
         """Возвращает список известных экспериментов с их статусами."""
```

Ruff (0.15.13)

[warning] Docstring contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF002)